x/wasm/keeper/api.go

```diff
@@ -4,11 +4,14 @@ import (
 	"fmt"
 
 	sdk "github.com/line/lbm-sdk/types"
+	"github.com/line/lbm-sdk/x/wasm/types"
 	wasmvm "github.com/line/wasmvm"
```

Comment:
This is old change. But I don't know why this use `wasmvm` alias. It is not necessary.

Reply:
Yes, it looks a little odd.
However, The wasmvm repo actually exports the go package as 'cosmwasm'. This is not our fix.
https://github.com/CosmWasm/wasmvm/blob/f7015565a59255cd09ebfcbf9345f3c87666fedd/lib.go#L1

So, if there is no alias, we should use 'cosmwasm.xxx'. However, since it is already used as an alias in another source file, I following it.
https://github.com/line/lbm-sdk/blob/d2ba9c174c2ef392865017dd98dcd7651e1f62d5/x/wasm/keeper/keeper.go#L15

Reply:
When I check this commit history, this was changed from `cosmwasm "github.com/CosmWasm/go-cosmwasm"` to `wasmvm "github.com/CosmWasm/wasmvm"`.  Maybe I think this is generated by IDE. Because Webstom sometimes add a alias equals with module name.

I thought this is modified in our side, but I check the commit history more deeply, I found this is Peters's change. So, it is ok.
Thank you.